proto/regen/ecocredit/v1alpha1/tx.proto

```diff
@@ -101,6 +101,13 @@ message MsgCreateBatchRequest {
   google.protobuf.Timestamp end_date = 6
       [(gogoproto.stdtime) = true, (gogoproto.moretags) = "yaml:\"end_date\""];
 
+  // project_location is the location of the project backing the credits in this
+  // batch. It is a string of the form
+  // <country-code>[-<sub-national-code>[ <postal-code>]], with the first two
```

Comment:
Should we add some text to clarify that sub national and postal codes are optional and can allow greater or lesser granularity?

Reply:
Yup, good point